examples/user_guide/04-Dimensioned_Containers.ipynb

```diff
@@ -97,11 +99,11 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Exploring a parameter space using widgets is one of the most flexible approaches but as we noted above it also makes it difficult to compare between different parameter values. HoloViews therefore supplies several container objects, which behave similarly to a ``HoloMap`` but have a visual representation:\n",
+    "Exploring a parameter space using widgets is one of the most flexible approaches but as we noted above it also makes it difficult to compare between different parameter values. HoloViews therefore supplies several container objects, which behave similarly to a ``HoloMap`` but have a different visual representation:\n",
```

Comment:
That makes much more sense! How about
'a visual representation showing the effect of all parameter values simultaneously'